packages/pockethost.io/src/components/ValidateUserToken.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { client } from '$src/pocketbase'
+  import { globalUserData } from '$util/stores'
+  import { onMount } from 'svelte'
+  import { getRouter } from '$util/utilities'
+  import publicRoutes from '$util/public-routes.json'
+
+  const { refreshUserData } = client
+
+  onMount(async () => {
+    try {
+      // First check if the user even has a token
+      const doesUserTokenExist = localStorage.getItem('pocketbase_auth')
+
```

Comment:
No sure if this is the place to bring it up, but the PB client also has an `client.auth.onChange` event or something like that. We should be refreshing our user data any time that is called as well.

I assume it may change in realtime if an admin edits the user record.